release.yaml

```diff
@@ -16,3 +16,15 @@ rancher-monitoring:
 rancher-monitoring-crd:
   - 101.1.0+up40.1.2
   - 100.2.0+up40.1.2
+rancher-aks-operator:
+  - 101.1.0+up1.0.8-rc1
+  - 101.1.0+up1.0.8-rc1
+rancher-aks-operator-crd:
+  - 101.1.0+up1.0.8-rc1
+  - 101.1.0+up1.0.8-rc1
+rancher-eks-operator:
+  - 101.1.0+up1.1.6-rc1
+  - 101.1.0+up1.1.6-rc1
+rancher-eks-operator-crd:
+  - 101.1.0+up1.1.6-rc1
+  - 101.1.0+up1.1.6-rc1
```

Comment:
duplicated lines?

Reply:
Ah no, I am bumping the versions of both aks and eks which each have 2 charts (a rancher-<kev2>-operator chart and a rancher-<kev2>-operator-crd chart)

Reply:
I mean, there are duplicated versions under each section, like two `101.1.0+up1.0.8-rc1` under `rancher-aks-operator`

Reply:
Perks of feeling under the weather today;) thanks